upgrade/db/postgresql/ezplatform-2.5.latest-to-3.0.0.sql

```diff
@@ -41,3 +41,23 @@ ALTER TABLE ezeditorialworkflow_markings
     ADD COLUMN reviewer_id INTEGER,
     ADD COLUMN result TEXT;
 --
+
+-- EZEE-2988: Added availability for schedule hide --
+BEGIN;
+ALTER TABLE ezdatebasedpublisher_scheduled_version RENAME COLUMN publication_date TO action_timestamp;
+ALTER TABLE ezdatebasedpublisher_scheduled_version ADD COLUMN action VARCHAR(32);
+UPDATE ezdatebasedpublisher_scheduled_version SET action = 'publish';
+ALTER TABLE ezdatebasedpublisher_scheduled_version ALTER COLUMN action SET NOT NULL ;
+COMMIT;
+--
+
+-- EZEE-2988: Added availability for schedule hide --
+BEGIN;
+ALTER TABLE public.ezdatebasedpublisher_scheduled_version
+    RENAME TO ezdatebasedpublisher_scheduled_entries;
+ALTER TABLE public.ezdatebasedpublisher_scheduled_entries
+    ALTER COLUMN version_id DROP NOT NULL;
+ALTER TABLE public.ezdatebasedpublisher_scheduled_entries
+    ALTER COLUMN version_number DROP NOT NULL;
```

Comment:
Missed also that, sorry: no need to explicitly specify schema, it's up to the maintainer (see that we don't use it anywhere).